views/partials/base.tmpl

```diff
@@ -42,15 +31,10 @@
 
 {{template "footer" .}}
 
-<!-- bower:js -->
-<script src="/bower_components/modernizr/modernizr.js"></script>
-<script src="/bower_components/jquery/dist/jquery.js"></script>
-<script src="/bower_components/bootstrap/dist/js/bootstrap.js"></script>
-<script src="/bower_components/moment/moment.js"></script>
-<script src="/bower_components/jquery-ui/jquery-ui.js"></script>
-<!-- endbower -->
+<script src="js/jquery-3.1.1.min.js"></script>
```

Comment:
Plausibly? But given the recent internet funtimes, I'd rather avoid it. There's not much benefit to it, especially as we don't have to worry about paying for disk space or bandwidth